src/mappings/nabla/backstopPoolEventHandler.ts

```diff
@@ -69,8 +75,18 @@ export async function handleBackstopPoolEvent(
 
 export async function handleBurn(
     ctx: EventHandlerContext,
+    event: Event_Burn,
     backstopPool: BackstopPool
 ) {
+    await createNablaSwapLiquidityWithdrawal(
```

Comment:
Shouldn't we call `createNablaBackstopLiquidityWithdrawal`? This one will save a `NablaSwapLiquidityWithdrawal` entity.